versioned_docs/version-v0.15/UserManuals/DORA.md

```diff
@@ -64,113 +66,123 @@ Calculating DORA metrics requires three key entities: **changes**, **deployments
 In the next section, we'll demonstrate how to configure DevLake to implement DORA metrics for the aforementioned example team.
 
 ### Collect GitHub & Jira data via `blueprint`
+
 1. Visit the config-ui at `http://localhost:4000`
-2. Create a `blueprint`, let's name it "Blueprint for DORA", add a Jira and a GitHub connection. Click `Next Step`
-![](https://i.imgur.com/lpPRZ6v.png)
+2. Create a project: `project1`. And then create a `blueprint`, let's name it "Blueprint for DORA", add a Jira and a GitHub connection. Click `Next Step`
```

Comment:
Text change:
Create a project: `project1`. Go to `project1` and create a `blueprint`...